### PR TITLE
Reorg peer connections periodically

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Reorg peer connections periodically
- Add numConnections to connectionStatus
- Simplify some grafana panels (by removing '5_m', '1_h' metrics)

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/809

Screen Shot 2021-12-23 at 2 47 32 PM

### CODE REVIEW

File: p2p/index.js [hunk 1]
```diff
@@ -250,14 +252,44 @@ class P2pClient {
...
+  async tryReorgPeerConnections() {
+    if (Object.keys(this.outbound).length < NodeConfigs.PEER_REORG_MIN_OUTBOUND) {
+      return;
+    }
+    if (this.steadyIntervalCount < NodeConfigs.PEER_REORG_STEADY_INTERVAL_COUNT) {
+      this.steadyIntervalCount++;
+    } else {
+      this.steadyIntervalCount = 0;
+      this.disconnectRandomPeer();
+      this.p2pState = P2pNetworkStates.EXPANDING;
```
Comment (liayoo) [inline p2p/index.js:281]: can we use updateP2pState() for this?

Comment (platfowner) [reply p2p/index.js]: Done, thanks!

**Comment by platfowner** [comment]
Comments addressed. PTAL~ @liayoo @minsulee2

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the review! Let me merge it now for fast field testing.. @liayoo @minsulee2

**Comment by liayoo** [comment]
> Thanks for the review! Let me merge it now for fast field testing.. @liayoo @minsulee2

Yay! I'll start [R11](https://www.notion.so/comcom/R11-91e9d19562c14bc19b9b730f3858d7e3) with these new changes right away.

**Review by minsulee2 — COMMENTED** [review]
I followed the code up! Thanks for the nice work!!

**Pull request merged**